src/oscar/templatetags/display_tags.py

```diff
@@ -42,29 +41,10 @@ def render(self, context):
 get_parameters = register.tag(get_parameters)
 
 
-@register.tag()
-def iffeature(parser, token):
-    nodelist = parser.parse(('endiffeature',))
+@register.filter
+def is_app_installed(app_label):
```

Comment:
I think this is better being a block template tag (e.g. `{% if_app_installed "reviews" %}`), like `iffeature` was, as filters are mostly used to transform the value.

Or at least a simple tag, which would be less code and more flexible, but require a bit more work to use:
```
{% is_app_installed "reviews" as reviews_app_is_installed %}
{% if reviews_app_is_installed %}
...
{% endif %}
```

Reply:
I would vote for the block tag, because it would make the templates a lot cleaner.